### PR TITLE
Fix for Issue A027 (Autocomplete No Options Text) and A045 (Phone Number regex)

### PR DESCRIPTION
**Pull request by andrewjknapp**
Added noOptionsText prop to Fields.DataSelect components within parentSelect, categorySelect, and schoolSelect functions in FormFormats.js. Also put {...props} on its own line in Fields.js line 79.

### CODE REVIEW

**Comment by dhuangg** [comment]
@jsflca2012 please review this when you can. I'll review after you

**Review by jsflca2012 — COMMENTED** [review]
Looks good, just remove console.log here

**Comment by andrewjknapp** [comment]
Thank you for catching that. I removed the console.log and the curly braces around the noOptionsText.

File: src/components/Form/FormFormats.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ const userMap = ({accountSearch}) => accountSearch.results.map(({user}) => ({
 
 const instructorSelect = (name) => (
     <Fields.DataSelect name={name} optionsMap={userMap}
-        request={SEARCH_INSTRUCTORS} noOptionsText={"No instructors available"}/>
+        request={SEARCH_INSTRUCTORS} noOptionsText="No instructors available"/>
```
Comment (dhuangg) [inline src/components/Form/FormFormats.js:72]: each attribute should be its own line. example:
```
<Fields.DataSelect name="name"
                   request={REQUEST}
                   noOptionsText="No options"
/> 
```

Comment (dhuangg) [reply src/components/Form/FormFormats.js:72]: This should be applied to everything

**Review by dhuangg — CHANGES_REQUESTED** [review]
Good changes - apply the last syntax change and we can merge this in

File: src/components/Form/FormFormats.js (hunk 1 above)
Comment (andrewjknapp) [reply src/components/Form/FormFormats.js]: Ok, added that formatting. Thanks

**Review by dhuangg — APPROVED** [review]
looks great! congrats on your first PR!

**Pull request merged**